scrapper/api/routes.py

```
@@ -0,0 +1,300 @@
+                "status": "completed"
+            }
+        )
+        
+        return IngestionResponse(
+            total=len(filter_result["jobs"]),
+            jobs=filter_result["jobs"]
+        )
+    
+    except HTTPException:
+        raise
+    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)